SCENE-SCRAPABLE.md

```diff
@@ -125,6 +136,8 @@ damnthatsbig.com|AMAMultimedia.yml|Gay
 danejones.com|RealityKingsOL.yml|
 darkx.com|GammaEntertainment.yml|
 daughterswap.com|PaperStreetMedia.yml|
+ddfbusty.com|DDFNetwork.yml|
+ddfnetwork.com|DDFNetwork.yml|
```

Comment:
ddfnetwork.com moved to pornworld.com

Reply:
pornworld.com have already is own Scraper (pornworld.yml)

Reply:
missed that you are right